sdk/scheduler/src/main/java/com/mesosphere/sdk/state/StateStoreUtils.java

```diff
@@ -210,35 +211,45 @@ public static void validateValue(byte[] value) throws StateStoreException {
      * Returns the current value of the 'suppressed' property in the provided {@link StateStore}.
      */
     public static boolean isSuppressed(StateStore stateStore) throws StateStoreException {
-        byte[] bytes = fetchPropertyOrEmptyArray(stateStore, SUPPRESSED_PROPERTY_KEY);
+        return fetchBooleanProperty(stateStore, SUPPRESSED_PROPERTY_KEY);
+    }
+
+    /**
+     * Sets a 'suppressed' property in the provided {@link StateStore} to the provided value.
+     */
+    public static void setSuppressed(StateStore stateStore, boolean isSuppressed) {
+        setBooleanProperty(stateStore, SUPPRESSED_PROPERTY_KEY, isSuppressed);
+    }
+
+    /**
+     * Returns the current value of the 'uninstall' property in the provided {@link StateStore}.
+     */
+    public static boolean isUninstalling(StateStore stateStore) throws StateStoreException {
+        return fetchBooleanProperty(stateStore, UNINSTALLING_PROPERTY_KEY);
+    }
 
+    /**
+     * Sets an 'uninstall' property in the provided {@link StateStore} to the provided value.
+     */
+    public static void setUninstalling(StateStore stateStore, boolean isUninstalling) {
```

Comment:
with the `false` case removed above, the boolean param here could be removed as well (always `true`). don't forget to update the javadoc